alfresco-mobile-android/google-services.json

```diff
@@ -0,0 +1,45 @@
+{
+  "project_info": {
+    "project_id": "alfresco-mobile",
+    "project_number": "1053450300169",
+    "name": "Alfresco Mobile"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:1053450300169:android:4c82694870595779",
+        "client_id": "android:org.alfresco.mobile.android.application",
+        "client_type": 1,
+        "android_client_info": {
+          "package_name": "org.alfresco.mobile.android.application"
+        }
+      },
+      "oauth_client": [],
+      "api_key": [],
+      "services": {
+        "analytics_service": {
+          "status": 2,
+          "analytics_property": {
+            "tracking_id": "UA-70658012-1"
```

Comment:
I'm not sure having this hardcoded is such a good idea - can it be parameterized via Bamboo?